diff --git a/flutter/tools/gn b/flutter/tools/gn
index fd193560..09200fb8 100755
--- a/flutter/tools/gn
+++ b/flutter/tools/gn
@@ -392,11 +392,11 @@
   out_dir = get_out_dir(args)
   command.append(out_dir)
   command.append('--args=%s' % ' '.join(gn_args))
-  print("Generating GN files in: %s" % out_dir)
+  print "Generating GN files in: %s" % out_dir
   try:
     gn_call_result = subprocess.call(command, cwd=SRC_ROOT)
   except subprocess.CalledProcessError as exc:
-    print("Failed to generate gn files: ", exc.returncode, exc.output)
+    print "Failed to generate gn files: ", exc.returncode, exc.output
     sys.exit(1)
 
   if gn_call_result == 0:
@@ -417,7 +417,7 @@
     try:
       contents = subprocess.check_output(compile_cmd_gen_cmd, cwd=SRC_ROOT)
     except subprocess.CalledProcessError as exc:
-      print("Failed to run ninja: ", exc.returncode, exc.output)
+      print "Failed to run ninja: ", exc.returncode, exc.output
       sys.exit(1)
     compile_commands = open('%s/out/compile_commands.json' % SRC_ROOT, 'w+')
     compile_commands.write(contents)
